WheelOfFortune/Game.cs

```diff
@@ -50,43 +50,29 @@ public class Game
         /// </summary>
         public void StartGame()
         {
-            Console.WriteLine("Press any button to start the game!");
-            Console.ReadKey(true);
-            AddPlayer();
+            Prompt.WelcomeMessage();
 
-            Random random = new();
-            int index = random.Next(0, allPuzzles.Count);
-            string nextPuzzle = allPuzzles[index];
-            Puzzle puzzle = new(nextPuzzle);
-            CurrentPuzzle = puzzle;
+            string person = Prompt.CreatePlayer();
```

Comment:
Thinking ahead to potential feedback we might receive - would it maybe be better for this variable to be named `name` or `playerName`?

Reply:
Agreed